libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405Mini/README.md

```diff
@@ -0,0 +1,97 @@
+# SpeedyBee F405 Mini Flight Controller
+
+The SpeedyBee F405 Mini is a flight controller produced by [SpeedyBee](http://www.speedybee.com/).
+
+## Features
+
+ - STM32F405 microcontroller
+ - ICM42688-P IMU
+ - DPS310 barometer
+ - 8Mb flash logging
+ - AT7456E OSD
+ - 6 UARTs
+ - 5 PWM outputs
+
+## Pinout
+
+![SpeedyBee F405 Mini](SpeedyBee_F405_Mini_Board.JPG "SpeedyBee F405 Mini")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (DJI-VTX, DMA-enabled)
+ - SERIAL2 -> UART2 (RX, DMA-enabled) 
+ - SERIAL3 -> UART3
+ - SERIAL4 -> UART4 (connected to internal BT module, not currently usable by ArduPilot)
+ - SERIAL5 -> UART5 (ESC Telemetry)
```

Comment:
should note RX pin only, on ESC connector

Reply:
how?where?

Reply:
No idea - I redid them